snaplet-acid-state.cabal

```diff
@@ -1,5 +1,5 @@
 name:           snaplet-acid-state
-version:        0.2.7
+version:        0.2.8
```

Comment:
Also, we don't need to bump `c` here.  If we don't raise the below lower bound we only need to bump this to `0.2.7.1`.